packages/scss/src/components/inputs/textfield/_textfield.scss

```diff
@@ -102,6 +102,35 @@
 
 	/* Display */
 
+	&.mod-togglePassword {
+
+		.textfield-suffix {
+			pointer-events: auto;
+		}
+
+		.textfield-input {
+			padding-right: _theme("spacings.bigger") !important;
+		}
+
+		.textfield-suffix-action {
+			text-decoration: none;
+			color: currentColor;
+			display: block;
+
+			&[aria-pressed="true"] {
```

Comment:
hmmm, this terminology traps us into a single way of handling this interaction: on-press display. We might decide to do a toggle display (toggle on click) rather than that. I think we should avoid too much dependency between style, function and interaction.

Reply:
No, we inherit the terminology from aria here. It's the one [dedicated to switches](https://inclusive-components.design/toggle-button/), and it doesn't imply to keep the button pressed.

Reply:
Ok I was afraid "pressed" meant being pressed